README.md

```diff
@@ -301,3 +301,103 @@ Pretty stnadard, we simply collect all the hashes and the associated types for t
 ### Evaluation
 
 As discusssed in the pinning section, evaluation is done on a fully typechecked and pinned `Expr`ession.
+
+## Developing Inferno scripts with VScode
+
+In a shell go to /inferno and run `nix build .#vscode-inferno-lsp-server`. Then in VScode press `ctrl+shift+P` and run `Install from VSIX`. In the window, navigate to /inferno/result and select `inferno-lsp-server.vsix`.
```

Comment:
Just for the sake of clarity, I might change `/inferno` to `~/inferno`

Reply:
I wrote it like this in case someone cloned anywhere besides home